src/Common/FileCache.cpp

```diff
@@ -512,13 +512,16 @@ void LRUFileCache::tryRemoveAll()
         auto & [key, offset] = *it++;
 
         auto * cell = getCell(key, offset, cache_lock);
+        if (!cell)
+           throw Exception(ErrorCodes::REMOTE_FS_OBJECT_CACHE_ERROR, "No cache cell for key: {}, offset: {}", keyToStr(key), offset);
```

Comment:
Ok, but then it would be a `ErrorCodes::LOGICAL_ERROR`, and the message should be `Cache is in inconsistent state: LRU queue contains entries with no cache cell`

Reply:
Agreed,I'll change the execution message.